### ISSUE TITLE
PCAS (PEPFuture) message from PaDS

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PaDS#21]
The PaDS portion of this message on Stage will look a bit odd to users (the part in parentheses, which comes from PaDS).

PCAS is the only journal that's in PEPFuture on Stage.

"""
This journal is in the process of being added to PEP-Web. This is content planned for the next release. It may be available on the publisher's website here. (Details of Item:PCAS can't be found on PaDS.)
"""

(A data update was recently run...it completed...not sure if it's a matter of PaDS in between updates or what.)

**Comment by jwoosnam** [comment]
From what I understood, and I thought was happening previously, all journals should be on pads even if they are future journals.  The journal PCAS is not on PaDS.
If I try https://stage-api.pep-web.org/v2/Metadata/Journals no PCAS is returned.

**Comment by nrshapiro** [comment]
@jwoosnam 

Yes, I was just thinking about that...yesterday, I made a change to the active journal flag so PEPFuture documents could be excluded from the statistics/journal count.  There must be a view on Stage I need to update.  Let me look into that.

**Comment by nrshapiro** [comment]
@jwoosnam 

Stage was updated last night, after fixing this issue (return of PCAS).  I assume the reload of PaDS info was called automatically at the end of that (good test to see if that's true.)  But FYI still getting the error this morning though.

<img width="528" alt="image" src="https://user-images.githubusercontent.com/4837554/150642142-cfaf4b5e-e12d-4bd8-aa1d-145914591b73.png">

**Comment by jwoosnam** [comment]
The content had not been reloaded.  I have started the job manually and
PCAS is now listed as future.
